Formula/c/copyparty.rb

```diff
@@ -0,0 +1,119 @@
+class Copyparty < Formula
+  include Language::Python::Virtualenv
+
+  desc "Portable file server"
+  homepage "https://github.com/9001/copyparty"
+  url "https://files.pythonhosted.org/packages/ab/fb/95b58ef0848ed78c80fec0e19749f8daccdb848d3dd4294d3ba8b0978845/copyparty-1.18.6.tar.gz"
+  sha256 "f136106ec36f3b91dcab3a473c039b4f2b999afe5c3a5a9edb5d217a4cfb0bd2"
+  license "MIT"
+
+  depends_on "cmake" => :build
+  depends_on "ninja" => :build
+  depends_on "pkgconf" => :build
+  depends_on "cryptography"
+  depends_on "python@3.13"
+  depends_on "vips"
+  depends_on "zeromq"
+
+  uses_from_macos "libffi"
+
+  resource "argon2-cffi" do
+    url "https://files.pythonhosted.org/packages/0e/89/ce5af8a7d472a67cc819d5d998aa8c82c5d860608c4db9f46f1162d7dab9/argon2_cffi-25.1.0.tar.gz"
+    sha256 "694ae5cc8a42f4c4e2bf2ca0e64e51e23a040c6a517a85074683d3959e1346c1"
+  end
+
+  resource "argon2-cffi-bindings" do
+    url "https://files.pythonhosted.org/packages/b9/e9/184b8ccce6683b0aa2fbb7ba5683ea4b9c5763f1356347f1312c32e3c66e/argon2-cffi-bindings-21.2.0.tar.gz"
+    sha256 "bb89ceffa6c791807d1305ceb77dbfacc5aa499891d2c55661c6459651fc39e3"
+  end
+
+  resource "cffi" do
+    url "https://files.pythonhosted.org/packages/fc/97/c783634659c2920c3fc70419e3af40972dbaf758daa229a7d6ea6135c90d/cffi-1.17.1.tar.gz"
+    sha256 "1c39c6016c32bc48dd54561950ebd6836e1670f2ae46128f67cf49e789c52824"
+  end
```

Comment:
Not needed; this is a dependency of `cryptography`.

Reply:
Is this not required for argon2-cffi? I saw it is requored for it.

Reply:
It might be, but my point is that it is already installed since it is a dependency of `cryptography`.

Reply:
Ah... That's it! Thank you.